woodwork/table_accessor.py

```diff
@@ -144,7 +145,12 @@ def init(self,
             if self._schema.time_index is not None:
                 self._sort_columns(already_sorted)
 
+        self.make_index = make_index
```

Comment:
Put this at the end of the `init` method in order to also have a value for when a schema is passed in. (I'm assuming we want to be able to have this value stored even when initializing with a schema)

It's a bit of a weird situation because we warn users against passing additional parameters like `make_index` when also passing a `schema`, and say that we ignore them, but now we're technically not ignoring `make_index` (though passing `make_index=True, schema=schema` won't impact the underlying index at all--in fact it'd error in parameter validation).

However, it wouldn't really be useful for users to be able to pass a `make_index` value when also passing a `schema` because we won't also let them set a new index value.

Reply:
Do you think it would make more sense to always set `self.make_index` to `False` (the default) inside the `if schema is not None` block, and then set it to the user supplied value in the `else` block? That way I think the behavior would be more consistent with our warning.

If that change is made, might be good to add a comment explaining why it is done that way.

Reply:
that seems like a good idea.... definitely more consistent with our warning.